decidim-initiatives/app/commands/decidim/initiatives/vote_initiative.rb

```diff
@@ -17,6 +17,7 @@ def initialize(form)
       # - :invalid if the form was not valid and we could not proceed.
       #
       # Returns nothing.
+
```

Comment:
Adding this line would break tools that show the comment for a method (and doesn't make sense adding this newline) 

```suggestion
```